lib/src/model.dart

```diff
@@ -221,8 +259,9 @@ class Accessor extends ModelElement
     implements EnclosedElement {
   GetterSetterCombo _enclosingCombo;
 
-  Accessor(PropertyAccessorElement element, Library library)
-      : super(element, library);
+  Accessor(
+      PropertyAccessorElement element, Library library, Member originalMember)
```

Comment:
I see some subclasses pass `null` in for `originalMember`; perhaps make it an `[optional]` param?